datumaro/datumaro/components/formats/voc.py

```diff
@@ -97,11 +102,104 @@ class VocPath:
     SUBSETS_DIR = 'ImageSets'
     IMAGE_EXT = '.jpg'
     SEGM_EXT = '.png'
+    LABELMAP_FILE = 'labelmap.txt'
 
     TASK_DIR = {
         VocTask.classification: 'Main',
         VocTask.detection: 'Main',
         VocTask.segmentation: 'Segmentation',
         VocTask.action_classification: 'Action',
         VocTask.person_layout: 'Layout',
-    }
\ No newline at end of file
+    }
+
+
+def make_voc_label_map():
+    labels = sorted(VocLabel, key=lambda l: l.value)
+    label_map = OrderedDict(
+        (label.name, [VocColormap[label.value], [], []]) for label in labels)
```

Comment:
Why `[VocColormap[label.value], [], []]`? Is it better to use OrderedDict with color, parts, actions keys?

Reply:
Better than what? I did not get the question.

Reply:
Better then the current approach. In the code I see indexes like [0], [1], [2]. Is it better to have ["color"], ["parts"], ["actions"] instead?